### PR TITLE
Added two additional parameters in order to build to proper signature:

### PR DESCRIPTION
**Pull request by pedgarcia**
--appid = identify the application id instead of hard-coded
--apikey = user's API KEY in order to build the proper signature

Both parameters are mandatory.

### CODE REVIEW

File: examples/mediafire-cli.py [hunk 1]
```diff
@@ -153,6 +151,10 @@ def main():  # pylint: disable=too-many-statements
                         default=os.environ.get('MEDIAFIRE_EMAIL', None))
     parser.add_argument('--password', dest='password', required=False,
                         default=os.environ.get('MEDIAFIRE_PASSWORD', None))
+    parser.add_argument('--appid', dest='appid', required=False,
```
Comment (roman-yepishev) [inline examples/mediafire-cli.py]: The script itself is part of "Coalmine" app, which has the APP_ID 42511. While overriding that may be useful to test other clients, forcing the user of the script to provide that value explicitly is not really user-friendly.

File: examples/mediafire-cli.py [hunk 2]
```diff
@@ -248,8 +250,8 @@ def main():  # pylint: disable=too-many-statements
 
     client = MediaFireClient()
 
-    if args.email and args.password:
-        client.login(args.email, args.password, app_id=APP_ID)
+    if args.email and args.password and args.appid and args.apikey:
+        client.login(args.email, args.password, app_id=args.appid, api_key=args.apikey)
```
Comment (roman-yepishev) [inline examples/mediafire-cli.py]: Client login should not depend on providing args.apikey. API key is optional.

Comment (pedgarcia) [inline examples/mediafire-cli.py]: Hm , I don't get it. When I was trying to use the mediafire-client not providing the API_KEY, I always got an error related to 'wrong signature' generated. As soon as I passed the key, it started to work. What am I missing?

File: examples/mediafire-cli.py (hunk 1 above)
Comment (pedgarcia) [inline examples/mediafire-cli.py]: How about having the parameter but defaulting it to the hardcoded id?

Comment (roman-yepishev) [inline examples/mediafire-cli.py]: :+1:

File: examples/mediafire-cli.py (hunk 2 above)
Comment (roman-yepishev) [inline examples/mediafire-cli.py]: Your application record on MediaFire side can be configured to not require api key:
screenshot from 2016-02-09 21-04-31

45211 has the Secret Key disabled, so it should work. Were you using it with a different application id?

Comment (pedgarcia) [inline examples/mediafire-cli.py]: Ah, gotcha. I wasn't aware I could disable the secret key.

Comment (pedgarcia) [inline examples/mediafire-cli.py]: I was using the app with my own APP_ID but I guess I don't need it to access my own drive information?

**Comment by pedgarcia** [comment]
Roman, I believe I messed up my fork. I will close this pull request, ok?

**Pull request closed without merging**